tests/ttnn/multichip_unit_tests/test_multidevice_TG.py

```diff
@@ -450,11 +474,22 @@ def test_galaxy_attn_matmul(M, N, head_dim, num_heads, cluster_shape, device_mes
         packer_l1_acc=True,
     )
 
+    if num_heads == 80:
+        core_grid = ttnn.CoreGrid(y=10, x=8)  # Why did we only use 40 cores here?
```

Comment:
is 10x8 valid grid? Do not we use one row for dispatch?

Reply:
Good point, Brian pointed out to me that we probably only have 8x8 available for compute even though we have 10x8 in general on a galaxy, I think it's then falling back to the 8x8. I'll double check and update.

Reply:
I verified, it's only running on a subset of cores, in this case 5x8.

Reply:
So, is there any point specifying different core grids, is this something the user should worry about, if metal is making the choice of subgrid?

Reply:
In this case we don't need to. I think this is a relic from the sharded version - but we might add that back in when we have sharding support for the line allgathers I think, so I'd rather leave it in because there we'd need to specify the grid to shard on.